engine/runtime-with-instruments/src/test/java/org/enso/interpreter/test/instrument/AvoidIdInstrumentationTagTest.java

```diff
@@ -38,7 +40,11 @@ public void initContext() {
             RuntimeOptions.LANGUAGE_HOME_OVERRIDE,
             Paths.get("../../distribution/component").toFile().getAbsolutePath()
         )
-        .logHandler(OutputStream.nullOutputStream())
+        .option(
+                RuntimeOptions.LOG_LEVEL,
+                Level.WARNING.getName()
+        )
+        .logHandler(System.err)
```

Comment:
Doesn't this pollute the `test` command output with a lot of debug messages? Ideally they should only be displayed when the test fails. There is no point in showing logged messages when the test succeeds, I think.

Reply:
Nope. I have not seen a lot of, if any, warnings being reported in our test suite.